torch/_inductor/pattern_matcher.py

```diff
@@ -50,8 +50,13 @@
 Constant = Any
 NodeOrConstant = Union[Constant, torch.fx.Node]
 
+
+class Multiple:
+    pass
+
+
 # Sentinel indicating multiple quantities can be matched
-MULTIPLE = object()
+MULTIPLE = Multiple()
```

Comment:
it's a little bit confusing that `Multiple` and `MULTIPLE` are now both in scope. 

Can you not use `MULTIPLE` the sentinel object as a type annotation ?

Reply:
No you can't. I also tried something like `Literal[object]`, but was also unable to get it to work.